### ISSUE TITLE
Running on Ubuntu produces error

### ISSUE THREAD

**Issue by tcby45** [Kwoth/NadekoUpdater#2]
Whenever I run the linux script on my Ubuntu instance (thanks, klincheR!), it returns the following:

```cs
Failed to load /home/botty/NadekoBot v0.8 b10.1/ubuntu.14.04-x64/publish/libcoreclr.so, error: libunwind.so.8: cannot open shared object file: No such file or directory
Failed to bind to coreclr
```
Neither running the included `nadekoupdater` script in the base directory or directly calling `publish/nadekoupdater` works; they both produce the above error.

**Comment by Kwoth** [comment]
Ok you can try 2 things:
1. 
- Follow the guide https://dotnet.github.io/getting-started/ to install dotnet
- Try running program again

If that doesn't work
2.
- Clone the repo
- `dotnet build` in the root folder of the project 
- run the exe that will be produced

**Comment by tcby45** [comment]
Sadly can't do either, as dotnet refuses to install on 15.10 due to a dependency not supporting it. Time for me to dig up klincheR's Linux tutorial and start anew

**Comment by Kwoth** [comment]
yep, sorry, just don't use updater then, just get the release and unzip it, thats pretty much what the updater does. (plus easy updates :\)
